civis/utils/_jobs.py

```diff
@@ -16,15 +15,11 @@ def run_job(job_id, api_key=None):
 
     Returns
     -------
-    results : :class:`~civis.polling.PollableResult` or
-    :class:`~civis.pubnub.SubscribableResult`
-        A `PollableResult` or `SubscribableResult` object if a Pubnub
-        connection is available.
+    results : :class:`~civis.futures.CivisFuture`
+        A `CivisFuture` object.
     """
     client = APIClient(api_key=api_key, resources='all')
     run = client.jobs.post_runs(job_id)
-    if 'pubnub' in client.feature_flags and has_pubnub:
-        return SubscribableResult(client.jobs.get_runs,
-                                  (job_id, run['id']),
-                                  api_key=api_key)
-    return PollableResult(client.jobs.get_runs, (job_id, run['id']))
+    return CivisFuture(client.jobs.get_runs,
+                       (job_id, run['id']),
+                       api_key=api_key)
```

Comment:
This can use `poll_on_creation = False`, correct? Since it's creating a job immediately before initializing the object, you know it hasn't finished.